src/libraries/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -236,6 +244,7 @@ public partial class SslStream : System.Net.Security.AuthenticatedStream
         public override void Write(byte[] buffer, int offset, int count) { }
         public override System.Threading.Tasks.Task WriteAsync(byte[] buffer, int offset, int count, System.Threading.CancellationToken cancellationToken) { throw null; }
         public override System.Threading.Tasks.ValueTask WriteAsync(System.ReadOnlyMemory<byte> buffer, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public System.Threading.Tasks.Task AuthenticateAsServerAsync( ServerOptionsSelectionCallback optionCallback, object? state, System.Threading.CancellationToken cancellationToken = default) { throw null; }
```

Comment:
The type is named Options, plural, but the parameter name is option, singular?

Reply:
I took it from the approved API but I think plural makes more sense. I will make the update unless somebody objects.